packages/asc-ui/src/components/Pagination/Pagination.tsx

```diff
@@ -32,16 +32,40 @@ export interface PaginationProps {
    * The number of pages to show in the pagination
    */
   paginationLength?: number
+  /**
+   * Label for previous
+   */
+  labelPrevious?: string
+  /**
+   * Label for next
+   */
+  labelNext?: string
+  /**
+   * Aria Label for previous
+   */
+  ariaLabelPrevious?: string
+  /**
+   * Aria Label for next
+   */
+  ariaLabelNext?: string
 }
 
 const DEFAULT_PAGINATION_LENGTH = 7
+const DEFAULT_LABEL_PREVIOUS = 'vorige'
+const DEFAULT_LABEL_NEXT = 'volgende'
+const DEFAULT_ARIA_LABEL_PREVIOUS = 'Vorige pagina'
+const DEFAULT_ARIA_LABEL_NEXT = 'Volgende pagina'
```

Comment:
These constants (DEFAULT_LABEL_... and DEFAULT_ARIA_LABEL_...) are only used once, right? You can add them directly as defaults on line 65-68 imo

Reply:
done!